nms/packages/magmalte/app/views/equipment/EnodebDetailConfig.js

```diff
@@ -34,9 +34,826 @@ import {colors, typography} from '../../theme/default';
 import {makeStyles} from '@material-ui/styles';
```

Comment:
Many of our files are too long - is there a neat way to split this up into two or more files for easier reading?

Reply:
I already modify these points